### DISCUSSION TITLE
Issue Templates on Mobile

### DISCUSSION THREAD

**Discussion by CuzImBisonratte** [community/community#15721] [Mobile] [answered]
Issue templates don't work on Mobile.

When clicking the button to create an issue, you see the list of templates, but when you click one it is an empty issue (so the template isn't loading.

**System Info:**
System: iPhone 8 
iOS-Version: 15.5 PB 3

**Comment by aconsuegra** [comment, marked as answer]
Hi @CuzImBisonratte, thanks for the feedback! We have identified the issue and are working right now on a fix that hopefully will be included in our upcoming release. Sorry about the inconvenience!
